docs/en/how_to_sync.md

```diff
@@ -0,0 +1,6 @@
+# How to sync by cloud apps
+It's easy to enable cloud syncing. Boostnote watches file changes, so file-based cloud services are best for sync.
+Move your Storage into your directory of any file-based cloud services by hand like this below.
+![command](https://raw.githubusercontent.com/BoostIO/Boostnote/docs/images/how_to_move.png)
```

Comment:
No, I cannot. I had tried. By the way, I'll close this PR. Because the wiki is better than `docs`.